### ISSUE TITLE
Multithreading camera settings change

### ISSUE THREAD

**Issue by neilthomson** [jim-easterbrook/python-gphoto2#40]
I have several cameras connected to one computer, and these cameras (Sony A5000) are very slow to change their settings and python_gphoto2 does it unthreaded so settings only change on one camera at a time, takes a while.

I have recompiled with gp_camera_set_config and gp_camera_set_single_config as threaded and it works fine with the Sonys at least. I can make a pull request if this can be merged in, unless there's a reason not to?

**Comment by jim-easterbrook** [comment]
I assume you mean adding this to camera.i:
```
%thread gp_camera_set_config;
%thread gp_camera_set_single_config;
```
I can't think why I haven't already done this. I suppose it wasn't painfully slow with the cameras I've used, unlike `gp_camera_get_config`. No need to bother with a pull request, I'll just do it.

**Comment by jim-easterbrook** [comment]
This is now released in v1.8.1
